sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -980,7 +980,8 @@ object SQLConf {
       .createWithDefault(true)
 
   val FILES_MAX_PARTITION_BYTES = buildConf("spark.sql.files.maxPartitionBytes")
-    .doc("The maximum number of bytes to pack into a single partition when reading files.")
+    .doc("The maximum number of bytes to pack into a single partition when reading files" +
+      " for data source table.")
```

Comment:
What if the built-in Parquet/ORC reader are used to process tables created by Hive(`spark.sql.hive.convertMetastoreParquet=true`)?

Reply:
Yes,  I have checked the configuration of the job I attached in JIRA.

We don't set spark.sql.hive.convertMetastoreParquet, so its value is true with default.

Reply:
I think @wangyum meant that, if we enable `spark.sql.hive.convertMetastoreParquet` or `spark.sql.hive.convertMetastoreOrc`, reading Hive table is also affected by this configurations. So, saying "for data source table" is not entirely correct.

"data source table" is also incorrect in the sense of DataFrame or Dataset.

Also, it does not support, IIRC, regular other data source tables. it only applies to file sources.

Reply:
I would simply describe this as, 

> ... partition when Spark file-based sources are used to read files

Reply:
thanks very much.